external/mmdetection/detection_tasks/apis/detection/__init__.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2021 Intel Corporation
+#
```

Comment:
Just as a discussion point.

1. As we now have common set of tasks/utils/whatever for all mmdetection-powered templates, it looks safe to remove extra level of nesting (`detection`) here. But probably not in this PR.
2. Having `mmdetection` directory containing `detection_tasks` makes the story a bit complicated. Maybe we should not reference any particular algo backend in the folders' tree of OTE at all and leave it being hidden under the neutral `submodule`? I foresee difficulties if we end up having multiple detection backends though. Any thoughts on this?

Reply:
1. makes sense
2. not sure that I fully understand, but "external" and "mmdetection" (and other) names are kept here because of CI. I made this transition so that all paths such as "external/mmdetection/init_venv.sh", "external/mmdetection/tests", etc are not changed. I guess these paths are hardcoded, @LeonidBeynenson , right? Or are you, @druzhkov-paul , telling about different things?

Reply:
You captured my point right.

I see, that we have limitation at the CI side. @LeonidBeynenson could you please estimate how difficult and time-consuming would that be to support modified directories structure? I mean 100% transition, not "almost completed" state.

At the same time do we have an agreement that such renaming is a reasonable thing to do? Or it does not pay off taking into account all of the required changes in the infrastructure?